doc/nrf/protocols/matter/end_product/attestation.rst

```diff
@@ -76,39 +82,45 @@ To read more about DCL and how to use it, see the :ref:`ug_matter_device_dcl` pa
 Providing device with Device Attestation data
 *********************************************
 
-To pass the Device Attestation procedure, a Matter device that wants to join a Matter fabric must include a set of information required for the Device Attestation procedure.
-This needs to be provided by the manufacturer.
+To pass the Device Attestation procedure, a Matter device that wants to join a Matter fabric must include two sets of information required for the Device Attestation procedure: factory data and Certification Declaration.
+Both need to be provided by the manufacturer.
 
 Getting preliminary Device Attestation data
 ===========================================
 
-To generate the required data, the manufacturer needs to get the following information from CSA:
+The manufacturer needs to get the following information from CSA:
 
-* Vendor ID (VID) - A unique 16-bit number that identifies the manufacturer.
+* Vendor ID (VID) - a unique 16-bit number that identifies the manufacturer.
   This can be obtained from CSA when the manufacturer becomes one of `Connectivity Standards Alliance members`_.
-* Certification Declaration (CD) - A cryptographic document created by the CSA for each device type, used to confirm that a given type of device was certified.
+* Certification Declaration (CD) - |matter_cd_definition|.
+  Until you obtain the CD from CSA, you can use a generic CD for testing purposes during the commissioning.
+
+After the manufacturer obtains Certification Declaration, it can be provided directly to the device, because it is not part of the factory data.
```

Comment:
I do not quite get this sentence. What does it mean directly to the device and why the reason for it is that it's not part of factory data? I would not compare CD to factory data at all and write something like this:

```suggestion
After the manufacturer obtains Certification Declaration, it can be provided to the device with the new firmware version, for example using the Device Firmware Upgrade functionality.
```